packages/terra-table/src/subcomponents/Cell.jsx

```diff
@@ -280,8 +286,14 @@ function Cell(props) {
 
   const CellTag = isRowHeader ? 'th' : 'td';
 
+  // Added to check if rowHeight is defined, it will take precedence. Otherwise the minimum row height would be used.
+  // Here 2.5rem is the default row minimum height.
```

Comment:
The default height comment should not be here.  This is an implementation detail of the parent component.  It might not be the same for other implementations that use cell.

Reply:
Done.
commit: 3173b8e8bc4563c520944f3ff6c29b90f3c9124c
Thanks